src/engine/ReluConstraint.h

```diff
@@ -186,7 +187,7 @@ class ReluConstraint : public PiecewiseLinearConstraint
     PhaseStatus getDirection() const;
 
 private:
-    unsigned _b, _f;
+    unsigned _b, _f, _layer;
```

Comment:
This is a bit problematic, or at least is a shift in paradigm. So far, a PL constraint has no notion of a DNN, and hence of a layer. This is also how we present things in the ReluPlex paper - it is a general tool for solving LRA queries with ReLUs. So far, the only place that contained the notion of a DNN was the NLR. 

Maybe lets have a setScore() function, and have the AcasParser or someone else invoke it for the ReLU constraints?

[There's another change of paradigm here, and that is that constraints rank their splitting priority as opposed to the SMT Core deciding for them, but I'm okay with that]

Reply:
I see. That makes sense. I added a setScore() method, and let the AcasParser invoke it if we use the EarliestReLU divide strategy.